tests/data/transcription/est00.txt

```diff
@@ -0,0 +1,5 @@
+0.120   0.290   225.000
```

Comment:
I made them up. It's a toy example that allows testing all the different aspects of the evaluation (onset matching, offset matching, pitch matching) and whose results can be computed by hand in order to unittest the transcription eval's final output.